crates/bevy_math/src/rects/rect.rs

```diff
@@ -277,6 +277,27 @@ impl Rect {
         r
     }
 
+    /// Check if this rectangle overlaps another rectangle.
+    /// Touching edges count as overlapping.
```

Comment:
"Overlap" means "fully contains" here, right? Not just "intersects somewhere" or "one side is overlapping". Maybe needs to be clarified.

Reply:
I would go with the known naming scheme of "intersects" and "contains"

Reply:
Hey both - thanks for the suggestions, actually that makes sense. I'll go for `contains`.